codegen/src/main/java/software/amazon/awssdk/codegen/poet/auth/scheme/DefaultAuthSchemeParamsSpec.java

```diff
@@ -108,34 +137,57 @@ private void addFieldsAndAccessors(TypeSpec.Builder b) {
                               .build());
 
         if (authSchemeSpecUtils.usesSigV4()) {
-            b.addField(FieldSpec.builder(String.class, "region")
-                                .addModifiers(Modifier.PRIVATE, Modifier.FINAL)
-                                .build());
+            if (authSchemeSpecUtils.usesSigV4()) {
```

Comment:
usesSigv4 checked twice

Reply:
I don't know how that happened, I will fix it.